### PR TITLE
Fix typo in docs about plus addressing

### PR DESCRIPTION
**Pull request by armandduijn**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/plus-addressing-in-exchange-online.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ You can’t enable plus addressing in the Exchange admin center (EAC); you can o
 You can create new plus addresses by adding a new tag. You can use plus addresses as unique addresses for services that you sign up for. 
 
 >[!NOTE]
-> Some web forms don’t support plus signs in email addresses. If you have subscribed to some email list subscription services using an SMTP email address, and you need to unsubscribe from them, you must use the email address that you subsribed with. You cannot unsubscribe by sending emails with plus address. You also cannot unsubscribe from some email subscriptions using unsubscribe email messages with plus addresses if you have subscribed to them. 
+> Some web forms don’t support plus signs in email addresses. If you have subscribed to some email list subscription services using an SMTP email address, and you need to unsubscribe from them, you must use the email address that you subscribed with. You cannot unsubscribe by sending emails with plus address. You also cannot unsubscribe from some email subscriptions using unsubscribe email messages with plus addresses if you have subscribed to them.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/plus-addressing-in-exchange-online.md:52]:
```suggestion
> Some web forms don’t support plus signs in email addresses. If you have subscribed to some email list subscription services using an SMTP email address, and you need to unsubscribe from them, you must use the email address that you subscribed with. You cannot unsubscribe by sending emails with plus addresses. You also cannot unsubscribe from some email subscriptions using unsubscribe email messages with plus addresses if you have subscribed to them. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - one more, please.

**Comment by armandduijn** [comment]
@JohanFreelancer9 Could you have another look at this PR? I've made your requested changes.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**